react/utils/index.tsx

```diff
@@ -188,3 +188,14 @@ export function orderSplitMessage({
     takeaways,
   })
 }
+
+export function parseBankInvoiceUrl(url: string) {
+  const isEncrypted = !!url.match(/(\*.\*.)+\*\w\*/g)
+
+  if (!isEncrypted) return url
+  if (!window || !window.location) return ''
+
+  return `/login?returnUrl=${encodeURIComponent(
```

Comment:
```suggestion
  return `${__RUNTIME__.rootPath || ''}/login?returnUrl=${encodeURIComponent(
```

Reply:
In order to access the runtime like this, I think you have to use the window. Like: `window.__RUNTIME__` or you could use the render HOC `withRuntimeContext`. I think the latter is more intuitive